apps/drec-api/src/pods/file/file.service.ts

```diff
@@ -87,6 +87,72 @@ export class FileService {
     });
   }
 
+  public async updateOrganization(
```

Comment:
I would rename this method, because it's not updating organization, it's assigning files to user organization